.github/workflows/integration_test_run.yaml

```diff
@@ -252,6 +222,11 @@ jobs:
             module="-k ${{ matrix.modules }}"
           fi
           tox -e integration -- --model testing --keep-models $series $module $args ${{ inputs.extra-arguments }}
+          mv $(find .tox/integration/tmp/pytest -name *.charm) .
```

Comment:
This relies on the fact that we run tox... it is very restrictive... 
Wondering if that's a good idea

Reply:
Well, the whole workflow relies on tox being run so it's not really much of a change